applications/add_unit_to_parameter_in_db.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/python3
+
+"""
+    Summary
+    -------
+    This application is used to add a unit to parameters in the DB.
+
+    This application should not be used by anyone but expert users and only in extreme cases.
```

Comment:
Suggest to replace `extreme` by `unusual`.

Reply:
Done